### PR TITLE
Fix linter warnings

### PR DESCRIPTION
**Pull request by beaconchain**
Fixes all warning raised by go-staticcheck

### CODE REVIEW

File: exporter/exporter.go [hunk 1]
```diff
@@ -75,7 +75,7 @@ func Start(client rpc.Client) error {
 			logger.Fatal(err)
 		}
 
-		for epoch := head.HeadEpoch - 1; epoch >= 0; epoch-- {
+		for epoch := head.HeadEpoch - 1; true; epoch-- {
```
Comment (qu0b) [inline exporter/exporter.go:78]: true here, is that right?

Comment (qu0b) [reply exporter/exporter.go:78]: there might be a chance that epoch goes below 0 when the client returns an error and continue is called

Comment (beaconchain) [reply exporter/exporter.go:78]: epoch is an uint hence can never go below 0

Comment (qu0b) [reply exporter/exporter.go:78]: https://go.dev/play/p/w3U4mMxbPCL

**Review by axljoker03 — APPROVED** [review]

**Pull request closed without merging**